tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -143,6 +154,51 @@ where
 		Ok(json!(json_value))
 	});
 
+	if let Some(state) = state {
```

Comment:
Maybe it's better to have the `if let` inside the handler? I.e. we should always have this rpc method available, just return empty/error when no state is present by the time of requesting